pkg/daemon/update.go

```diff
@@ -512,6 +512,16 @@ func (dn *Daemon) update(oldConfig, newConfig *mcfgv1.MachineConfig) (retErr err
 	return dn.finalizeAndReboot(newConfig)
 }
 
+/// removeRollback removes the rpm-ostree rollback deployment.  It
+/// takes up space, and we don't generally expect administrators to
+/// use this versus e.g. removing broken configuration.  We only
+/// remove the rollback once the MCD pod has landed on a node, so
+/// we know kubelet is working.
```

Comment:
oops, gotta replace `///` with `//`

Reply:
Context switching between Rust and Go is hard :smile: